File: java/ArmStrong.java
import java.util.Scanner;
public class ArmStrong {
    public static void main(String[] args) {
        int rem,count=0;
        int ans = 0;
        Scanner sc = new Scanner(System.in);
        System.out.println("enter any number");
        int n = sc.nextInt();
        int temp=n;
        while (n>0){
          rem=n%10;
          ans=ans+(rem*rem*rem);
          n=n/10;
      }
if(temp==ans)
        System.out.println(temp+" is a Armstrong number ");
else
    System.out.println(temp+"  is not a Armstrong number");
    }
    }

File: Armstrong.c
#include<stdio.h>

// Checking if a number is Armstrong or not

int main()
{
    int n, sum = 0, temp, rem;
    printf("Enter any 3-digits number: ");
    scanf("%d", &n);
    temp = n;

    while(n > 0)
    {
        rem = n % 10;
        sum = sum + rem * rem * rem;
        n = n / 10;
    }

    if(temp == sum) {
        printf("The number %d is an Armstrong number!", temp);
    }
    else {
        printf("The number %d is not an Armstrong number!", temp);
    } 

    return 0;
}
